checkov/secrets/plugins/load_detectors.py

```diff
@@ -77,6 +78,8 @@ def add_detectors_from_code(custom_detectors: List[Dict[str, Any]], code: str, s
                 secret_policy['title'],
                 check_id,
                 regex,
+                # Only one prerun per multiline regex
+                code_dict['definition'].get('prerun', [''])[0],
```

Comment:
since we're going to load it from a json, let get it safely

Reply:
everywhere up we are using this a if it's None the linter fails the tests :|